apps/portal/examples/postgrest-template/content/source/Dockerfile.migrate

```
@@ -0,0 +1,28 @@
+
+# Install wget for downloading golang-migrate
+RUN apk add --no-cache wget
+
+# Download golang-migrate binary
+RUN wget -qO- https://github.com/golang-migrate/migrate/releases/download/v4.17.0/migrate-linux-amd64.tar.gz | tar xz -C /tmp/
+
+# Final stage: minimal runtime image
+FROM alpine:3.19
+
+# Install PostgreSQL client for schema inspection if needed
+RUN apk add --no-cache postgresql-client bash
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)